src/mark.ts

```diff
@@ -72,18 +77,6 @@ export interface TooltipContent {
   content: 'encoding' | 'data';
 }
 
-/** @hidden */
-export type Hide = 'hide';
```

Comment:
FYI, the old hide mode was a secret experimental feature (not part of public API) that I experimented with many years ago.  It's not documented and thus can be removed.